week1/homework/src/server.js

```diff
@@ -9,7 +9,39 @@ function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    const handleRequest = (statusCode, jsonMessage) => {
```

Comment:
You should not define functions inside of other functions. `handleRequest` here gets defined every time we call `createServer`.